modules/nf-core/minimap2/align/minimap2-align.diff

```diff
@@ -0,0 +1,23 @@
+Changes in module 'nf-core/minimap2/align'
+--- modules/nf-core/minimap2/align/main.nf
++++ modules/nf-core/minimap2/align/main.nf
+@@ -2,15 +2,13 @@
+     tag "$meta.id"
+     label 'process_medium'
+ 
+-    // Note: the versions here need to match the versions used in the mulled container below and minimap2/index
+-    conda "bioconda::minimap2=2.24 bioconda::samtools=1.14"
++    conda 'bioconda::minimap2=2.21 bioconda::samtools=1.12'
+     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+         'https://depot.galaxyproject.org/singularity/mulled-v2-66534bcbb7031a148b13e2ad42583020b9cd25c4:1679e915ddb9d6b4abda91880c4b48857d471bd8-0' :
```

Comment:
The conda versions are changed heer but not the container, maybe have a look at this